web/app.fluidity.money/app/routes/$network/dashboard.tsx

```diff
@@ -500,6 +501,15 @@ export default function Dashboard() {
               Recieve
             </GeneralButton>
             */}
+            {(isTablet || isMobile) && showMobileNetworkButton && (
+              <>
```

Comment:
Hey so the way that I read the message Ivan on discord, it sounded like we only want the `<ChainSelectorButton>` to be in the navbar for the mobile+tablet view, but not for the regular webapp. Is this what you're referring to, or are you point out something else?

https://discord.com/channels/845326901437399071/1087908628729700373/1087921683568001044

Reply:
@greenbookwebb nah I just mean the `<ChainSelectorButton />` component is wrapped in a react fragment `<></>` which isn't needed since there's only 1 element